lib/controllers/issueish-searches-controller.js

```diff
@@ -93,7 +94,9 @@ export default class IssueishSearchesController extends React.Component {
         this.props.workingDirectory,
       ),
       {pending: true, searchAllPanes: true},
-    );
+    ).then(() => {
```

Comment:
Hmm...maybe the reporter should have an async helper for recording async events. 🤔

Reply:
Possibly! Personally, I think using `addEvent` like this is just fine -- it's clear and readable. That said, definitely open to trying it out and seeing how it feels / if there are additional benefits to using an async helper.